tests/e2e/pageobjects/ide/Terminal.ts

```diff
@@ -151,13 +143,13 @@ export class Terminal {
             `//li[contains(@id, 'shell-tab-terminal') or contains(@id, 'shell-tab-plugin')]` +
             `//div[@class='p-TabBar-tabLabel']`;
 
-        const terminalTabs: WebElement[] = await this.driverHelper.waitAllPresence(By.xpath(terminalTabTitleXpathLocator));
+        const terminalTabs: WebElement[] = await this.driverHelper.waitAllPresence(By.xpath(terminalTabTitleXpathLocator), TimeoutConstants.TS_SELENIUM_CLICK_ON_VISIBLE_ITEM);
```

Comment:
Is it necessary to pass the variable here, when the same is defined as a default in the method itself?

Reply:
Fixed.